includes/Block/Web_Stories_Block.php

```diff
@@ -87,10 +87,34 @@ public function register() {
 			$this->get_script_settings()
 		);
 
+		// Use path to javascript directory or if javascript not build, then fallback to dev location.
+		$base_path = $this->get_base_path( 'assets/js/block.json' );
+		if ( ! $base_path ) {
+			$base_path = $this->get_base_path( 'assets/src/web-stories-block/block/block.json' );
+		}
```

Comment:
Since it's marked as a required file in `web-stories.php`, it does not make sense to me to load from `assets/src`

Reply:
Then this could be simplified to

```php
register_block_type_from_metadata(
$this->assets->get_base_path('assets/js/block.json',
...
)
```

No need for the separate `register_block_type()` method anymore

Reply:
So couple of things here. It wasn't originally another method, but then the PHPMD saw a long method and kicked off. So I broke it up. 

I have also returned the check to see if the `'assets/js/block.json' file exists, so we don't have to build JS in our PHP unit test runs.